<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Columns</class>
 <widget class="QDialog" name="Columns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Columns</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="ckInstanceId">
     <property name="text">
      <string>Instance ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckHardwareId">
     <property name="text">
      <string>First hardware id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckManufacturer">
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckProvider">
     <property name="text">
      <string>Provider</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckClass">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ckInstanceId</tabstop>
  <tabstop>ckHardwareId</tabstop>
  <tabstop>ckManufacturer</tabstop>
  <tabstop>ckProvider</tabstop>
  <tabstop>ckClass</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Columns</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
